weave-and-docker-platform/1-machine.md

```diff
@@ -1,150 +1,233 @@
+Check to see that all worked well:
 
-As we will use this container interactively, we will need to attach it first with `docker attach` and be sure to hit
-return/enter key to get the prompt.
+~~~bash
+weave status
+~~~
 
-    > docker $DOCKER_CLIENT_ARGS attach pinger
+and then check the {{ weaveproxy }} logs from Docker to see that proxy is running:
 
-First, let's ping the other container by it's DNS name
+~~~bash
+docker logs weaveproxy
+~~~
 
-    pinger:/# ping -c3 pingme.weave.local
-    PING pingme.weave.local (10.128.0.1): 56 data bytes
-    64 bytes from 10.128.0.1: seq=0 ttl=64 time=0.158 ms
-    64 bytes from 10.128.0.1: seq=1 ttl=64 time=0.161 ms
-    64 bytes from 10.128.0.1: seq=2 ttl=64 time=0.062 ms
+Now you are ready to deploy containers and also use DNS so that the containers can discover each other.
 
-    --- pingme.weave.local ping statistics ---
-    3 packets transmitted, 3 packets received, 0% packet loss
-    round-trip min/avg/max = 0.062/0.127/0.161 ms
+### Deploy Two Containers to the Weave Network
 
-Great, now let's see if it responds on TCP port 4000
+The first app to be deployed is called `pingme`. It consists of a simple netcat (aka `nc`) server running on TCP port 4000, which sends a short message, `Hello, Weave!` to any client that connects to it.
 
-    pinger:/# echo "What's up?" | nc pingme.weave.local 4000
-    Hello, Weave!
-    pinger:/#
+~~~bash
+> docker create --name=pingme gliderlabs/alpine nc -p 4000 -lk -e echo 'Hello, Weave!'
+~~~
 
-We can exit the test container now.
+The second containerized app is called `pinger`, and it will be launched interactively using the `-ti` flag, where the container can acccept and run few simple commands.
 
-    pinger:/# exit
+~~~bash
+> docker create --name=pinger -ti gliderlabs/alpine sh -l
+~~~
 
-And, as all worked well, let's get rid of both containers by running
+Start the containers:
 
-    > docker $DOCKER_CLIENT_ARGS rm -f pingme pinger
-    pingme
-    pinger
+~~~bash
+> docker start pinger
+> docker start pingme
+~~~
 
-Next, we could probably do something a bit more fancy, but for that we will need to introduce a few more concepts in the
-[next chapter][ch2].
+Check to see that {{ weavedns }} has registered them:
+
+~~~bash
+weave status
+~~~
+
+### Interacting with Containerized Apps
+
+First ping one of the containers using `docker exec` command:
+
+~~~bash
+> docker exec -i pinger ping -c3 pingme.weave.local
+~~~
+
+~~~bash
+PING pingme.weave.local (10.128.0.1): 56 data bytes
+64 bytes from 10.128.0.1: seq=0 ttl=64 time=0.100 ms
+64 bytes from 10.128.0.1: seq=1 ttl=64 time=0.114 ms
+64 bytes from 10.128.0.1: seq=2 ttl=64 time=0.111 ms
+
+--- pingme.weave.local ping statistics ---
+3 packets transmitted, 3 packets received, 0% packet loss
+round-trip min/avg/max = 0.100/0.108/0.114 ms
+~~~
+
+Test if pinger responds on TCP port 4000 as expected:
+
+~~~bash
+docker exec -i pinger echo "What's up?" nc pingme.weave.local 4000
```

Comment:
Actually this was corrected by Matthias.  I was getting a 'node not found error' when I ran this, and he said it was because of incorrect scope? 

But ok, I will change it back.